MailCore/Cache/AccountManager/AccountManager.swift

```diff
@@ -259,6 +259,7 @@ public final class AccountManager: RefreshTokenDelegate, ObservableObject {
             if mailbox.isLimited {
                 mailbox.quotas = try await apiFetcher.quotas(mailbox: mailbox)
             }
+            mailbox.externalMailInfo = try await apiFetcher.externalMailFlag(mailbox: mailbox)
```

Comment:
As we have 3 calls to await here (permissions, quotas and external), would it be possible to use the `async let` syntax?
We can try together if you want!